src/Symfony/Component/HttpKernel/Tests/Controller/ArgumentResolver/RequestPayloadValueResolverTest.php

```diff
@@ -137,9 +137,9 @@ public function testQueryNullableValueArgument()
         $validator->expects($this->never())
             ->method('validate');
 
-        $resolver = new RequestPayloadValueResolver(new Serializer(), $validator);
+        $resolver = new RequestPayloadValueResolver(new Serializer([new ObjectNormalizer()]), $validator);
 
-        $argument = new ArgumentMetadata('valid', RequestPayload::class, false, false, null, true, [
+        $argument = new ArgumentMetadata('valid', QueryPayload::class, false, false, null, true, [
```

Comment:
Not sure I understand this change. Shouldn’t it be a separate test instead?

Reply:
this was my assumption from test name `testQueryNullableValueArgument` that `QueryPayload` should be used instead of `RequestPayload`